README.md

```diff
@@ -149,6 +150,8 @@ end
 
 The module skips the `verify_authenticity_token` before_action and adds an action to verify that the webhook came from Shopify.
 
+The WebhooksManager uses ActiveJob, if ActiveJob is not configured then by default Rails will run the jobs inline. However it is highly recommended to configure a proper background processing queue like sidekiq or resque.
```

Comment:
Maybe change to "like sidekiq or resque **in production.**"? I always just use the inline processor during development.